CHANGES.md

```diff
@@ -1,4 +1,12 @@
 
+## Version 2.2.2
+
+Released: 2024/10/7
```

Comment:
You don't need to add the actual date here. I will update it when it's been released.